### ISSUE TITLE
Space Bug - JTL Civilian Protection Guild - Corvette Mission - Imp Side

### ISSUE THREAD

**Issue by Namsemag** [SWG-Source/dsrc#45]
The final mission for the Civilian Protection Guild, when flying as Imperial is bugged. The corvette doe not get marked as disabled once you disable it.
I have repeated this 3 times now:
1. Kill all escorts first, then the corvette - Bug Occurs
2. Ignore the escorts - Bug Occurs
3. Go Overt so they are immediately hostile - Bug Occurs

No errors shown on server console and thus far, no work around found.

**Comment by IrishDarkshadow** [comment]
https://swg.fandom.com/wiki/Civilian_Protection_Guild

**Comment by Namsemag** [comment]
Same result flying as Neutral Pilot

**Comment by Namsemag** [comment]
![spacebug](https://user-images.githubusercontent.com/55237094/64874932-e71b7500-d643-11e9-860a-2a024cf19730.png)

**Comment by IrishDarkshadow** [comment]
spacequest/recovery/ep3_red_tie_recovery

**Comment by IrishDarkshadow** [comment]
I've just completed the mission without running into the problem. I killed the escorts then stripped the Corvette guns (overkill, I know) took out the shields, secondary shields then engine. At that point I got the message that I'd disabled the target. A short sequence took place where an imp operative (chiss fella) boarded and repaired the corvette. I then ran the escort mission to completion, went back to the Imp space station and collected my red tie interceptor deed.

![1](https://user-images.githubusercontent.com/12293802/64882836-cfe58300-d655-11e9-8152-94a33c6592db.jpg)
![2](https://user-images.githubusercontent.com/12293802/64882844-d3790a00-d655-11e9-9a57-43b19190b44e.jpg)
![3](https://user-images.githubusercontent.com/12293802/64882852-d96eeb00-d655-11e9-98f5-536626824540.jpg)
![4](https://user-images.githubusercontent.com/12293802/64882854-dd027200-d655-11e9-8179-e3e50b9dc127.jpg)
